.github/workflows/builder_image_golang.yaml

```diff
@@ -25,7 +33,7 @@ jobs:
           sudo apt install zstd golang docker.io containerd runc
       - name: Docker build
         run: |
-          docker build -f ./images/builder-golang/Dockerfile -t ${GITHUB_REGISTRY}/${BUILDER_IMAGE_NAME}:${VERSION} .
+          docker build -f ./distrib/images/builder-golang/Dockerfile -t ${GITHUB_REGISTRY}/${BUILDER_IMAGE_NAME}:${VERSION} .
```

Comment:
What does distrib mean?

Reply:
I understand your goal, but this is not enough IMHO, `components` maybe?

Reply:
distribution, it will be.